README.md

```diff
@@ -61,7 +61,7 @@ api._api_key = "MY_API_KEY"
 
 ### DD_SITE
 
-If `DD_FLUSH_TO_LOG` is set to `false` (not recommended), and your data need to be sent to the Datadog EU site, you must set `DD_SITE` to `datadoghq.eu`. Defaults to `datadoghq.com`.
+If `DD_FLUSH_TO_LOG` is set to `false` (not recommended), you must set `DD_SITE`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.
```

Comment:
If there's a default value, then `must` is the wrong word here. May? Should?

Reply:
The must, in this case, sounds like the right word since it applies to the specific set of conditions.